src/main/kotlin/com/pubnub/api/PubNub.kt

```diff
@@ -174,6 +184,8 @@ class PubNub internal constructor(
         executorService
     )
 
+    private fun accepts(envelope: AnnouncementEnvelope<*>) = true
```

Comment:
you can move this to line 173 simply as the second argument: `{ true }`

Reply:
ok, I will move it.